static/src/javascripts/projects/common/modules/atoms/youtube-player.spec.ts

```diff
@@ -34,8 +31,10 @@ jest.mock('lib/config', () => ({
 	}),
 }));
 
-jest.mock('@guardian/consent-management-platform', () => ({
-	onConsentChange: jest.fn((callback: Callback) =>
+jest.mock('@guardian/libs', () => ({
+	// eslint-disable-next-line -- ESLint doesn't understand jest.requireActual
+	...jest.requireActual<typeof import('@guardian/libs')>('@guardian/libs'),
```

Comment:
Why are we skipping the linting rule, here? ESLint [definitely supports `jest.requireActual`](https://github.com/jest-community/eslint-plugin-jest/blob/293737f64344f818be5d8dab1902b383356ebd21/docs/rules/no-untyped-mock-factory.md)